main.go

```diff
@@ -60,6 +60,8 @@ func main() {
 	showLatestStable := getopt.StringLong("show-latest-stable", 'S', defaultLatest, "Show latest implicit version. Ex: tfswitch --show-latest-stable 0.13 prints 0.13.7 (latest)")
 	latestFlag := getopt.BoolLong("latest", 'u', "Get latest stable version")
 	showLatestFlag := getopt.BoolLong("show-latest", 'U', "Show latest stable version")
+	latestRequiredFlag := getopt.BoolLong("latest-required", 'r', "Get latest required by module version")
```

Comment:
What's this argument use case? Ain't this the default behavior of `tfswitch` when no cmdline args are provided?

Reply:
Aha, got it. So it's sort of an enforcement/explicitness of what does `case checkTFModuleFileExist(*chDirPath) && len(args) == 0:` conditional. Makes sense.
Should probably use `installTFProvidedModule(*chDirPath, &binPath, mirrorURL)` then (prefix `binPath` with `&`) at line 125 in `main.go` in this PR.

Reply:
> prefix binPath with &

Well, `custBinPath` is already a pointer to string, and `binPath` is just the dereferenced value, so they are the same:
```
	case fileExists(TOMLConfigFile) || fileExists(HomeTOMLConfigFile):
		version := ""
		binPath := *custBinPath
```

Reply:
My point is about consistency across the same codebits.

Reply:
Aha, I got your point. We're talking about different "consistencies" =) I'm basing mine off of the use of `installTFProvidedModule()` for `checkTFModuleFileExist()` case within the same `case fileExists(TOMLConfigFile) || fileExists(HomeTOMLConfigFile)` block.

Though... it looks like `binPath` vs `custBinPath` logic is messed up and is a bit convoluted within the `case fileExists(TOMLConfigFile) || fileExists(HomeTOMLConfigFile):` conditional. Especially given the comment that `custBinPath` should override TOML config value.
So seemingly the logic intended should've been something like `if custBinPath == defaultBin; then read TOML config; else binPath = custBinPath`, but it is vice versa and hence some nested conditional cases use `custBinPath`, while others use `bibPath` 😲 Or I'm missing something essential… 🤔

@ikorchynskyi @jukie @warrensbox What do you guys think of last paragraph above ⬆️ re the intended logic — should we rectify this or it's all fine and @ikorchynskyi should keep using `custBinPath` for `latestRequiredFlag` case (see blow)? https://github.com/warrensbox/terraform-switcher/blob/cb690d56cee6c724a4af38fc222b02c14afb9d46/main.go#L123-L125